docs/articles/high-availability.md

```diff
@@ -0,0 +1,40 @@
+# Deploying a Highly Available zot Registry
+
+> :point_right: A highly available zot registry can be easily implemented using zot's registry synchronization feature.
+
+In the zot configuration, the `sync` extension allows a zot instance to mirror another zot instance with various container image download policies, including on-demand and periodic downloads. You can use the zot `sync`  function combined with a load balancer such as [HAProxy](https://www.haproxy.com) to implement a highly available registry.
+
+Two failover configurations are possible:
+
+* Active/active
+
+    Registry requests are load-balanced between two zot instances, each of which mirrors the other.
+
+* Active/standby
+
+    Registry requests are sent by the load balancer to the active zot instance, while a standby instance mirrors the active. If the load balancer detects a failure of the active instance, it then sends requests to the standby instance.
+
+> :pencil2: The highly available zot registry described in this article differs from [zot clustering](clustering.md). Although zot clustering provides a level of high availability, the instances share common storage, whose failure would affect all instances. In the method described in this article, each instance has its own storage, providing an additional level of safety.
+
+For details of configuring the `sync` extension, see [OCI Registry Mirroring With zot](mirroring.md).
+
+## Configuring an active/active registry
+
+An active/active zot registry can be implemented between two zot instances by configuring the `sync` extension in each instance to point to the other instance.  In this scheme:
+
+* a load balancer such as HAProxy or a [DNS-based routing](https://coredns.io/plugins/loadbalance/) scheme is deployed for round-robin load balancing between zot instances
```

Comment:
Furthermore, one has to manage traffic split (path based routing) so some traffic goes to one instance and rest goes to another.
https://www.haproxy.com/blog/path-based-routing-with-haproxy

Reply:
Is this to keep a repo from being fragmented across two instances?